### PR TITLE
Add EIP: Prevent Block Gas Smuggling

### PR DESCRIPTION
**Pull request by benaadams**
Prevent the circumvention of the block gas limit by excluding certain gas discounts and refunds from block gas accounting, except for those reflecting actual reductions in computational work. 

This proposal aims to align the block gas usage with the Ethereum Virtual Machine's (EVM) real workload, preventing "block gas smuggling." It complements [EIP-7623](./eip-7623) by further reducing the maximum block size and its variance, enhancing network stability and efficiency.

### CODE REVIEW

File: EIPS/eip-7776.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+eip: 7776
```
Comment (abcoathup) [inline EIPS/eip-7776.md:2]:
```suggestion
eip: 7778
```
Assigning next sequential EIP/ERC/RIP number.  7776 was already assigned to an ERC

Please also update the filename.

File: EIPS/eip-7776.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+title: Prevent Block Gas Smuggling
+description: Exclude Discounts & Refunds for Block GasLimit Enforcement
+author: Ben Adams (@benaadams)
+discussions-to: https://ethereum-magicians.org/t/eip-7776-prevent-block-gas-smuggling/21234
```
Comment (abcoathup) [inline EIPS/eip-7776.md:6]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-7778-prevent-block-gas-smuggling/21234
```
I updated the EIP number in Eth Magicians

File: EIPS/eip-7776.md (hunk 1 above)
Comment (benaadams) [reply EIPS/eip-7776.md]: Renamed filename

**Comment by g11tech** [comment]
A small query otherwise looks gtm

File: EIPS/eip-7778.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+eip: 7778
+title: Prevent Block Gas Smuggling
+description: Exclude Discounts & Refunds for Block GasLimit Enforcement
+author: Ben Adams (@benaadams)
+discussions-to: https://ethereum-magicians.org/t/eip-7778-prevent-block-gas-smuggling/21234
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-10-01
+requires: 7623
```
[the remark this answers is not in the document]
Comment (benaadams) [reply EIPS/eip-7778.md]: Removed

**Review by g11tech — APPROVED** [review]

**Pull request merged**